src/assets/styles/diablo-global.css

```diff
@@ -44,8 +44,7 @@ abbr {
             user-select: none; /* Non-prefixed version, currently
                                   supported by Chrome, Edge, Opera and Firefox */
 }
-.sr-only,
-table caption {
```

Comment:
Vue 3 tables don't allow for <caption>. Instead of using a visually-hidden <caption>, use `aria-label` on the <table>.